pkgs/applications/misc/gollum/gemset.nix

```diff
@@ -4,10 +4,10 @@
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
-      sha256 = "0k3lijm4dmiz977bfmpclk5glj5jwv7bidamwwwywm60ywb0n4n4";
```

Comment:
I just rerun your update script (which is very nice to have btw) and I'm getting some more recent versions of the Gems. Could you run it one more time?

Reply:
done :+1: